specs/catalog-karapace.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/karapace/config/resolve/schema/id/retry/zilla.yaml

```
@@ -0,0 +1,46 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+telemetry:
+  exporters:
+    exporter0:
+      type: test
+      options:
+        events:
+          - qname: test.catalog0
+            id: catalog.karapace.unretrievable.schema.id
+            message: UNRETRIEVABLE_SCHEMA_ID 9
+          - qname: test.catalog0
+            id: catalog.karapace.retrievable.schema.id
+            message: RETRIEVED_SCHEMA_ID 9
+catalogs:
+  catalog0:
+    type: karapace
+    options:
+      url: http://localhost:8081
+bindings:
+  net0:
+    type: test
+    kind: server
+    options:
+      catalog:
+        catalog0:
+          - id: 9
+      catalog-assertion:
+        - schema: null
+        - schema: '{"fields":[{"name":"id","type":"string"},{"name":"status","type":"string"}],"name":"Event","namespace":"io.aklivity.example","type":"record"}'
```

This looks great!

Suggest we generalize the concept of `assertions` as we'll potentially need that later for vault, etc.

```suggestion
      assertions:
        catalog:
          catalog0:
            - schema: null
            - schema: '{"fields":[{"name":"id","type":"string"},{"name":"status","type":"string"}],"name":"Event","namespace":"io.aklivity.example","type":"record"}'
```